### ISSUE TITLE
inference 

### ISSUE THREAD

**Issue by Part-Work** [AaronComo/LipFD#13]
is there any code to inference for one video? 

I want to check video to detect it is fake or real??

**Comment by twilight2001** [comment]
yes very interested, so far no luck adapting validate.py the results quite strange, even on AVlips (150 random selected mp4) :(

**Comment by twilight2001** [comment]
but validation performance is >0.95 accuracy after 50% data, excellent job !
